.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "stylelint.config": { "extends": "./client/.stylelintrc" }
+  "stylelint.config": { "extends": "./client/.stylelintrc" },
+  "eslint.nodePath": "./client/node_modules"
```

Comment:
Yea - I haven't looked, but there must be a way to get an editorconfig to adhere to this.